src/VASL/build/module/map/ASLDiceOverlay.java

```diff
@@ -723,25 +641,25 @@ private void RebuildSingleDieFaces()
                 mar_objSingleDieImage[l_i] = ColorChanger.changeColor(l_objImage, Color.white, m_clrSingleDieColor);
             }
         }
-        catch (Exception ex) 
+        catch (Exception ex)
         {
             ex.printStackTrace();
         }
     }
-    
+
     public boolean RegisterForDiceEvents(boolean bAdd)
     {
-        if ((!m_bRegisteredForDiceEvents) && (bAdd))
+        if (!m_bRegisteredForDiceEvents && bAdd)
         {
-            if (GameModule.getGameModule().getChatter() instanceof ASLChatter) 
+            if (GameModule.getGameModule().getChatter() instanceof ASLChatter)
             {
-                if (bAdd) 
+                if (bAdd)
                 {
                     ((ASLChatter) (GameModule.getGameModule().getChatter())).addListener(this);
-                    
+
                     if (m_objClock == null)
                     {
-                        m_objClock = new Timer (1000, this);
+                        m_objClock = new Timer(1000, this);
                         m_objClock.start();
                     }
                     else
```

Comment:
Note that the `else` block is unreachable, because `bAdd` was already checked and found true two scopes out from here. This is clearly a bug, but I have no idea what the intent was, so I'm leaving it alone.